extensions/database/module/scripts/index/database-import-controller.js

```diff
@@ -309,7 +309,7 @@ Refine.DatabaseImportController.prototype._getPreviewData = function(callback, n
             result.rowModel = data;
             callback(result);
           },
-          "jsonp"
+          "json"
```

Comment:
Some parts of the code were still making calls to the `get-rows` command with JSONP, so I have used this opportunity to clean that up.